beacon-chain/p2p/peers/status.go

```diff
@@ -120,8 +123,9 @@ func (p *Status) Add(record *enr.Record, pid peer.ID, address ma.Multiaddr, dire
 		return
 	}
 	peerData := &peerdata.PeerData{
-		Address:   address,
-		Direction: direction,
+		Address:        address,
+		Direction:      direction,
+		FirstAddedTime: timeutils.Now(),
```

Comment:
Do we also need to make sure that `store.PeerDataGetOrCreate()` method sets `FirstAddedTime` to `timeutils.Now()`?